### DISCUSSION TITLE
Does the recursion depth setting work correctly in windows 10?

### DISCUSSION THREAD

**Discussion by jarekt** [pkolaczk/fclones#228] [Q&A]
If folders are specified as the input paths to search, when on `-d 0`, the program does nothing, and on `-d 1` searches the folders including **all subdirectories**. Is this the correct behavior? Is this a Windows-only issue/bug? I'd imagine `-d 1` should search the input folders and not go into subdirectories.

**Comment by jarekt** [comment]
bump

**Comment by xgoisern** [comment]
Hello,
I just checked on a Linux Ubuntu system: the behaviour is the same as on Windows. It seems impossible to limit the search to the input path; Either fclones does nothing ( -d 0) or it explores also the first level of sub-directories (-d1) ...
Do I miss soemthing?

**Reply by jarekt** [reply]
I'm pretty sure it is just a bug. The entire project appears to be quite buggy and poorly documented, except for the simplest use cases. The author also appears to be quite inactive. There are issues reporting bugs that are multiple years old that he hasn't even responded to yet.
